crates/forge/tests/data/trace_resources/src/lib.cairo

```diff
@@ -21,7 +21,8 @@ fn use_builtins_and_syscalls(empty_hash: ClassHash, salt: felt252) -> ContractAd
     get_block_hash_syscall(1).unwrap_syscall();
     starknet::deploy_syscall(empty_hash, salt, array![].span(), false).unwrap_syscall();
     emit_event_syscall(array![1].span(), array![2].span()).unwrap_syscall();
-    send_message_to_l1_syscall(10, array![20].span()).unwrap_syscall();
+    send_message_to_l1_syscall(10, array![20, 30].span()).unwrap_syscall();
+    send_message_to_l1_syscall(10, array![20, 30].span()).unwrap_syscall();
```

Comment:
Why increase the number here? It influences all of the tests needlessly and makes the logic even more messy

Reply:
I just wanted to check if a few messages will be collected, but probably if the one is the rest will be too